outline/cheatsheet.md

```diff
@@ -88,12 +88,34 @@
 (defn function-name
   "description of function, optional"
   [param1 param2]
-  ((function-body)))
+  (function-body))
 ```
 
 * Functions that return true or false--called predicates--usually end in ?
 * map and reduce - Functions that take other functions
 
+## Map and Reduce
+
+```clj
+; apply a function to every element in a collection
+(map inc [1 2 3 4])
+;=> (2 3 4 5)
+; Similar to [(inc 1) (inc 2) (inc 3) (inc 4)]
+```
+**`map`** takes a function and a collection, then applies the function to each element in the collection in order. It returns the new collection.
```

Comment:
I'd drop "in order". When dealing with pure functions, one shouldn't be concerned with the order in which the function is applied to the elements.